include/pm/device_runtime.h

```diff
@@ -109,13 +109,31 @@ int pm_device_put(const struct device *dev);
  * @retval Errno Negative errno code if failure.
  */
 int pm_device_put_sync(const struct device *dev);
+
+/**
+ * @brief Wait on a device to finish an operation.
```

Comment:
Can we clarify what kind of operation ?

Reply:
Yep, will document it better. Operation here refers to put/get.